src/abacus-ts/calculators/markets.ts

```diff
@@ -79,18 +79,27 @@ function calculateDerivedMarketDisplayItems(market: IndexerWsBaseMarketObject) {
   };
 }
 
+function calculatePriceChangePercent(
+  priceChange24H: string | null | undefined,
+  oraclePrice: string | null | undefined
+): BigNumber | null {
+  if (priceChange24H == null || oraclePrice == null) {
+    return null;
+  }
+
+  const basePrice = MustBigNumber(oraclePrice).minus(priceChange24H);
+  return basePrice.gt(0) ? MustBigNumber(priceChange24H).div(basePrice) : null;
```

Comment:
super-nit: basePrice is kind of vague. I think it can be more descriptive as 
- priceYesterday, oldPrice, price24hAgo, etc.